src/serac/numerics/functional/domain_integral_kernels.hpp

```diff
@@ -5,16 +5,38 @@
 // SPDX-License-Identifier: (BSD-3-Clause)
 #pragma once
 
+#include "serac/serac_config.hpp"
 #include "serac/infrastructure/accelerator.hpp"
 #include "serac/numerics/functional/quadrature_data.hpp"
 #include "serac/numerics/functional/function_signature.hpp"
 #include "serac/numerics/functional/differentiate_wrt.hpp"
 
-#include <RAJA/index/RangeSegment.hpp>
-#include <RAJA/RAJA.hpp>
 #include <array>
 #include <cstdint>
 
+namespace {
```

Comment:
This should go into accelerator.hpp/cpp. We should also augment it to work with MPI and all cuda devices.